src/discordtipbot/discordbot.cr

```diff
@@ -120,6 +121,20 @@ class DiscordBot
         end
       end
     end
+
+    # warn users that the tipbot shouldn't be used as wallet if their balance exceeds @config.high_balance
+    spawn do
+      Discord.every(6.hours) do
+        users = @tip.get_high_balance(@config.high_balance)
+        next unless users
+        next if users.nil?
+
+        users.each do |x|
+          next unless x
```

Comment:
The array should never have a `nil` element.

Reply:
Compiler complained about `x` being `nil` here without that line